packages/kit/src/core/build/index.js

```diff
@@ -174,7 +174,10 @@ async function build_client({
 		plugins: [
 			svelte({
 				extensions: config.extensions,
-				emitCss: !config.kit.amp
+				emitCss: !config.kit.amp,
+				compilerOptions: {
+					hydratable: config.kit.ssr
```

Comment:
should we use `config.kit.hydrate` instead?
https://kit.svelte.dev/docs#configuration-hydrate

Reply:
good point, somehow i missed that option. should it be `!!(config.kit.ssr && config.kit.hydrate)` ?

without ssr there is nothing to hydrate, right?

Reply:
I believe that even without SSR you can still do hydration actually. Basically what it means is we send down the JSON blob with the initial response and then you build the page from that. It saves you from having to do a `fetch` to get the data

Reply:
switched to `!!config.kit.hydrate`